docker/docker-compose.yml

```diff
@@ -20,15 +20,15 @@ services:
       - LISK_DB_USER=${ENV_LISK_DB_USER}
       - LISK_DB_PASSWORD=${ENV_LISK_DB_PASSWORD}
       - LISK_DB_NAME=${ENV_LISK_DB_DATABASE}
+      - LISK_FILE_LOG_LEVEL=none
```

Comment:
Shall we get these vars from then environment?

Reply:
Those are environment variables (defined here, will be set inside the container)